### ISSUE TITLE
Wrong Highlighting and linting

### ISSUE THREAD

**Issue by berkaey** [microsoft/typescript-styled-plugin#85]
<img width="801" alt="screen shot 2018-12-06 at 09 26 41" src="https://user-images.githubusercontent.com/4539951/49565730-3310ef00-f939-11e8-8b64-6e2b70b6e5b7.png">

as you can see above plugin doesn't correctly highlight inside calc() and raises wrong errors when prettier split them into multiple lines.

**Comment by berkaey** [comment]
here you go...
```
export const AlphaPicker = styled.div.attrs(({ rgb = {} }) => {
  return { style: { background: `rgba(${rgb.r},${rgb.g},${rgb.b},${rgb.a})` } };
})`
  width: ${_`size.advanced.picker``px`};
  height: ${_`size.advanced.picker``px`};
  margin-left: calc(${_`size.advanced.picker``px`} / 2 * -1);
  margin-top: calc(
    ${_`size.advanced.picker``px`} / 2 * -1 + (${_`size.advanced.bar``px`} / 2)
  );
  border-radius: ${_`size.advanced.picker``px`};
  border: 2px solid #fff;
  cursor: default !important;
  user-select: none;
  box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.2);
`;
```

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)

After updating to that fork, follow up in the styled-components repo if this is still an issue
